### PR TITLE
Add meeting minutes for Feb 2023

### LINKED ISSUE #32: February 2023 Meeting
Last meeting we decided to set the date of the next meeting. The following is the information:

**Date:** Thursday 23rd of February 8pm UTC - 9pm UTC (convert to your timezone: https://www.timeanddate.com/worldclock/converter.html?iso=20230223T200000&p1=1440)
**Meeting Location**:  https://bbb.sfconservancy.org/b/chr-pm5-cl8-ctl
**Scribe:** @elplatt 
**Agenda:**
- https://github.com/ocapn/ocapn/issues/27
- Announcing new website for the group [ocapn.org](https://ocapn.org)
- #29 

If someone is able to setup up as a scribe, please let me know. If people have specific items they wish to add to the agenda please let me know in the comments.

### PR DESCRIPTION
**Pull request by tsyesika**
Fix #32

### CODE REVIEW

File: meeting-minutes/2023-02-23.md [hunk 1]
```diff
@@ -0,0 +1,217 @@
...
+
+zenhack: +1
+
+dale: +1
+
+markm: +1
+
+alan: +1
+
+## Approved: minutes from last week
```
Comment (ocdtrekkie) [inline meeting-minutes/2023-02-23.md:59]: s/week/month

File: meeting-minutes/2023-02-23.md [hunk 2]
```diff
@@ -0,0 +1,217 @@
...
+
+# TOPIC: OCapN URIs
+
+[https://github.com/ocapn/ocapn/issues/29](https://github.com/ocapn/ocapn/issues/29)
+
+cwebber: proposes Jessica continues to chair for this meeting. general agreement
+
+Jessica: in spritely, were working on changing URI format in both guile and racket versions of goblins. want input from this group while things are still easy to change. part of the NLnet grant is to address uris, seems like a good place to start
+
+cwebber: q+
```
Comment (ocdtrekkie) [inline meeting-minutes/2023-02-23.md:125]: Probably delete this?

**Review by ocdtrekkie — COMMENTED** [review]
I did have to duck out partway through the meeting. Did the meeting terminate abruptly due to time? The notes don't seem to look like any conclusion in the URI discussion was reached. It might be worth noting how the discussion was ended: Has it been tabled for continuation next month? Or have we covered everything we wanted to cover?

**Comment by zenhack** [comment]
@ocdtrekkie, yeah we didn't come to a tidy conclusion on the URI question. Nor did we explicitly plan on what to do with the issue.

**Review by zenhack — APPROVED** [review]

File: meeting-minutes/2023-02-23.md (hunk 1 above)
Comment (tsyesika) [reply meeting-minutes/2023-02-23.md]: Fixed, thanks!

File: meeting-minutes/2023-02-23.md (hunk 2 above)
Comment (tsyesika) [reply meeting-minutes/2023-02-23.md]: Fixed, thanks!

**Review by ocdtrekkie — APPROVED** [review]

**Comment by ocdtrekkie** [comment]
@dckc It still takes a month at present. If there's objections in between, we'll address it during the next meeting, otherwise it will just get merged. Jessica has been merging, as acting chair, I suspect our new chair will probably gain rights to this repo to do it as well.

The key benefit to approving this isn't necessarily about speeding merging... everyone can read the draft minutes here. But it saves a good 10% or more of the meeting's duration previously wasted voting on the minutes.

**Comment by dckc** [comment]
> everyone can read the draft minutes here

harrumph. It's kinda obscure for folks just browsing. And I often want to hand out a pointer to the meeting notes a few days after the meeting. Should I use... the PR? The file on a branch? A version of the file at a specific hash?

I'm spoiled by the [W3C 48hr norm](https://www.w3.org/2021/Process-20211102/#meeting-schedules), I suppose.

**Comment by ocdtrekkie** [comment]
Yeah, but we're three days into this being posted and the only people who have even approved it so far are from the Cap'n Proto/Sandstorm contingent.

File-on-branch is probably the best way to ensure it's both easily readable/formatted, and updated as the PR gets tweaked: https://github.com/tsyesika/ocapn/blob/meeting-2023-02/meeting-minutes/2023-02-23.md

**Comment by dckc** [comment]
I suppose the 48hr norm is _draft_ meeting notes. This is close enough.

**Comment by jar398** [comment]
Under rules I've seen elsewhere the minutes get posted for wide review (here that would mean in repo not just in issue) and then are approved by the group at the following business meeting (usually without discussion, but the placeholder is procedurally important).  That seems to work pretty well.

Any objections to my merging this PR?  (asking because there was some pushback here AND this would be my first merge in this repo.)

**Comment by ocdtrekkie** [comment]
My impression was we were using approval on the PR to indicate acceptance of the minutes. (I am not sure if the exact technical choice wasn't stated, but the agreed method according to the vote was for the approval to take place on the PR.) As of this time, the only individuals who have reviewed and approved the minutes on this PR are from the Sandstorm/Cap'n Proto group, which would not be a quorum.

Which is to say, as I approved the minutes, I do not object to the PR being merged, but I am not sure procedurally that is what was agreed to.

**Comment by ocdtrekkie** [comment]
> ## PROPOSED: Review meeting minutes out of band on PRs on the github PR, they will be considered approved by the next scheduled meeting provided group members have not provided blocking feedback.

I guess what we agreed to here, is that the PR should remain open until the next scheduled meeting regardless, and merged unless there's blocking feedback.

I think if we want to merge it sooner, that'd be great, but we should have at least one representative of each group indicating approval of the minutes via the review system. So far, this month's minutes are approved by Ian and myself (Sandstorm/Cap'n Proto), and last month's minutes, pre-merge, were approved by Ian and myself, Dan (Agoric), and while not approved persay by a Spritely representative, a Spritely representative merged it... which is kinda acceptance I suppose.

**Comment by zenhack** [comment]
That matches my memory of what we discussed: leave open until the next meeting, address any feedback as it comes in, and if there is no blocking feedback by the next meeting, it gets merged then and not discussed.

I don't think having this sit open for a month is actually a problem, and IMO it beats chasing people down.

**Comment by jar398** [comment]
@tsyesika Should I assign this issue to you, since you are supervising the PR?  The actions are (1) add requested changes if any (by amending the PR I presume), (2) merge the PR by the time of the March meeting.  These can only be done by someone with repo access.

**Pull request merged**